mhs/common/mhs_common/messages/tests/test_ebxml_error_envelope.py

```diff
@@ -0,0 +1,56 @@
+import os
+from pathlib import Path
+from xml.etree import ElementTree
+from unittest import TestCase
+
+from utilities.file_utilities import FileUtilities
+
+from mhs_common.messages.ebxml_error_envelope import EbxmlErrorEnvelope
+
+
+class TestEbxmlEnvelope(TestCase):
+    message_dir = Path(os.path.dirname(os.path.abspath(__file__))) / 'test_messages'
+
+    def test_is_ebxml_empty(self):
+        message = FileUtilities.get_file_string(Path(self.message_dir) / 'ebxml_response_error_empty.xml' )
```

Comment:
I doubt you need to do `Path(self.message_dir)` as `self.message_dir` is a `Path` already. eg I think this line could be changed:
```suggestion
        message = FileUtilities.get_file_string(self.message_dir / 'ebxml_response_error_empty.xml' )
```

and same for other lines in this file.

Reply:
Changed